### PR TITLE
Add support for public access modifiers

### PR DESCRIPTION
**Pull request by winston-riley-zocdoc**
this PR 
- Adds access modifiers for var and struct declaration.
- Drive by: if struct is empty render on single line

### CODE REVIEW

File: swift/struct.go [hunk 1]
```diff
@@ -3,23 +3,37 @@ package swift
 import "fmt"
 
 type StructDecl struct {
-	name    string
-	members []Declaration
+	name, accessModifier string
```
Comment (chris-smith-zocdoc) [inline swift/struct.go]: nit: can you put this on a new line, I find it easier to read

File: swift/struct.go [hunk 2]
```diff
@@ -28,8 +42,7 @@ func (s *StructDecl) AddMembers(members ...Declaration) *StructDecl {
 	return s
 }
 
-func NewStruct(name string) *StructDecl {
-	return &StructDecl{
-		name: name,
-	}
+func (s *StructDecl) Public() *StructDecl {
```
Comment (chris-smith-zocdoc) [inline swift/struct.go:46]: only public?

File: swift/var.go [hunk 3]
```diff
@@ -3,10 +3,15 @@ package swift
 import "fmt"
 
 type VarDeclaration struct {
-	name, typeName string
-	initValue      Writable
+	name, typeName, accessModifier string
```
Comment (chris-smith-zocdoc) [inline swift/var.go]: nit: newline

File: swift/struct.go (hunk 2 above)
Comment (winston-riley-zocdoc) [reply swift/struct.go:46]: eh, good call, i was being lazy, only adding what i need right now. But I might as well add other modifiers too.

**Review by vito-laurenza-zocdoc — APPROVED** [review]

**Pull request merged**